packages/wonder-blocks-tooltip/components/tooltip-bubble.js

```diff
@@ -37,12 +37,50 @@ export type Props = {|
 
     /** The `TooltipContent` element that will be rendered in the bubble. */
     children: React.Element<typeof TooltipContent>,
+
+    onActiveChanged: (active: boolean) => mixed,
+
+    /* A callback for updating the timeoutID refrence in tooltip  */
+    onTimeoutChanged: (timeoutID: ?TimeoutID) => mixed,
+
+    /* The timeoutID attached to the tooltip anchor */
+    anchorTimeoutID: ?TimeoutID,
+
     // TODO(somewhatabstract): Update react-docgen to support spread operators
     // (v3 beta introduces this)
     ...TooltipBubbleProps,
 |};
 
-export default class TooltipBubble extends React.Component<Props> {
+type State = {|
+    active: boolean,
+|};
+
+export default class TooltipBubble extends React.Component<Props, State> {
+    state = {
+        active: false,
+    };
+
+    _updateActiveState(hovered: boolean) {
```

Comment:
I think this function can be removed or merged with `_setActiveState` b/c both functions are being called in one place, right?